### ISSUE TITLE
torch.nn.MultiMarginLoss：equation format error

### ISSUE THREAD

**Issue by yeahQing** [pytorch/pytorch#58634]
## 📚 Documentation
The error is in torch.nn.MultiMarginLoss
> For each mini-batch sample, the loss in terms of the 1D input x and scalar output y is:

`loss(x, y) = ∑ max(0,margin−x[y]+x[i]))^p / x.size(0)`

An extra parentheses is added to the right side of the equation
 
​	

cc @brianjo @mruberry @albanD @jbschlosser

**Comment by soulitzer** [comment]
Thank you for reporting this issue! We would accept a fix for this.

**Comment by aflah02** [comment]
Hey @soulitzer I'm new here, can I work on this?

**Comment by mruberry** [comment]
@aflah02 Of course you may; anyone can work on any issue (whether a fix will be accepted is a different story...)

**Comment by aflah02** [comment]
@mruberry Thanks! I've made a pull request and mentioned this issue there
